pulsar/connection/descriptor.json

```diff
@@ -1,18 +1,43 @@
 {
 	"name": "pulsar-connection",
-	"type": "flogo:connection",
+	"type": "flogo:connector",
```

Comment:
i thought the type should be flogo:connection @fcastill does the ui validate the type?

Reply:
the alias type is registered as connection in the core also, so we should be consistent. https://github.com/project-flogo/core/blob/ed3ab0e51e4221be1fc0eef37d6f6d8e8fc566c0/support/connection/config.go#L102

Reply:
@mellistibco: Its "type": "flogo:connector" in MongoDB. https://github.com/project-flogo/datastore-contrib/blob/master/mongodb/connection/descriptor.json

Reply:
we should change it there also then :)

Reply:
@wnichols Lets make this change and give a try. :)

Reply:
If I change the OSS descriptor for the connection to "flogo:connection" the merge code will not generate a connection.json 
This is more likely to be a merge problem than anything else.

Reply:
what do you mean "the merge code will not generate a connection.json"?

Reply:
When the runtime and descriptors are imported to the enterprise build there is a merge process.  That process generates the required connector.json, activity.json etc files.  That merge process isn't working if the descriptor.json uses flogo:connection.
I'll look into it when I get some time.  Time is a problem for me at the moment :)

Reply:
I've addressed the upstream problem with the flogo-cli and changed the type to connection from connector here.  All working.